charters/MetadataSchema/charter.md

```diff
@@ -0,0 +1,63 @@
+
+# Metadata Schema
+
+## Description
+
+The HCA Metadata schema committee creates and maintains the Human Cell Atlas metadata schema in JSON format. These schema are designed to capture and provide structure for the descriptive scientific metadata associated with HCA datasets. These schemas aim to ensure the FAIRness* of the HCA data [The FAIR Guiding Principles for scientific data management and stewardship](https://www.nature.com/articles/sdata201618). 
+
+## Objective
+
+1. Ensure that HCA data are findable and the schemas are computationally queryable.
+2. Ensure that HCA metadata supports reproducibility and reusability.
+3. Ensure that HCA metadata has a consistent representation and is computationally queryable.
+4. Define JSON schema which enable objectives 1 and 2.
+5. Specify the entity structure for HCA metadata schema.
```

Comment:
I think this is what I would consider to be the HCA core domain model (and have used this terminology in the infrastructure charter which I am writing). If so, this is something that needs to be carefully negotiated across components as software has dependencies on this. I would suggest making it explicit that this is a negotiation, and we should review for consistency once I have published the PR for ingest infrastructure

Reply:
Can you propose a phrasing for the negotiation as I think 2, 3 and 4 are clearly in the metadata domain but 4 seems to be the thing which is more grey as to who is responsible for is?

Reply:
I have edited the line about entity models to try and reflect this.

Reply:
@tburdett can you confirm you are happy with the edited phrasing